src/OAuth2.php

```diff
@@ -357,6 +364,7 @@ public function __construct(array $config)
             'signingAlgorithm' => null,
             'scope' => null,
             'additionalClaims' => [],
+	    'partConsent' => '',
```

Comment:
whitespace is off here
Also, let's not use abbreviations. "partialConsent" is a perfectly fine variable name
```suggestion
	        'partialConsent' => '',
```

Reply:
I will change it to "enable_serial_consent" as suggested